action.yml

```diff
@@ -28,12 +28,16 @@ inputs:
     required: false
   lycheeVersion:
     description: "Use custom version of lychee link checker"
-    default: v0.18.0
+    default: v0.18.1
     required: false
   output:
     description: "Summary output file path"
     default: "lychee/out.md"
     required: false
+  checkbox:
+    description: "Add Markdown Styled Checkboxes to the output"
+    default: false
```

Comment:
I wonder if we should make this the default. Maybe that's what the majority of users would want? I don't see any downside right now. 🤔